integration_test/KeySequenceJJTest.re

```diff
@@ -42,9 +42,13 @@ runTest(
     );
     // TODO: Figure out why this check is failing...
     wait(~name="Validate buffer is empty", (state: State.t) => {
-      Model.Selectors.getActiveBuffer(state)
-      |> Option.map(Core.Buffer.getLines)
-      |> Option.map(Array.to_list) == Some([""])
+      let actual =
+        Model.Selectors.getActiveBuffer(state)
+        |> Option.map(Core.Buffer.getLines)
+        |> Option.map(Array.to_list);
+
+      // TODO
+      actual == Some([]);
     });
```

Comment:
The first TODO is form before this refactor. Not sure if this has anything to do with the change in behaviour here, but this also leads me to believe that there's something slightly off with the initialization of the initial buffer.

Reply:
Are you worried about this one in any way, or should we consider it a non-blocker?

Reply:
We can consider this a non-blocker - thanks for checking 👍